packer/mint-build.json

```diff
@@ -112,7 +113,8 @@
   "post-processors": [
     {
       "type": "checksum",
-      "output": "{{user `output_dir`}}/{{user `vm_name`}} {{user `semester`}}.{{.ChecksumType}}sum"
+      "checksum_types": "sha256",
+      "output": "{{user `output_dir`}}/image-{{user `semester` | lower }}.{{.ChecksumType}}sum"
```

Comment:
Can this be

```
"output": "{{user `output_dir`}}/{{user `output_filename`}}.{{.ChecksumType}}sum"
```

Seems like if it can, Ubuntu would need the same change. Don't know Packer well so just ignore if not doable.

Reply:
I don't think so, looks like `output_filename` is a parameter not a variable